misc/alert_rules.json

```diff
@@ -514,5 +514,9 @@
   {
     "rule": "devices.os = \"qnap\" && sensors.sensor_type = \"diskSmartInfo\" && sensors.sensor_current != \"0\"",
     "name": "QNAP NAS has a bad SMART disk status"
-  }
+  },
+  {
+    "rule": "UNIX_TIMESTAMP(NOW())- UNIX_TIMESTAMP(devices.last_polled) > (select config_value from config WHERE config_name=\"rrd.heartbeat\")",
```

Comment:
This needs a default value (coalesce) because rrd.heartbeat is not always set in the database.  Also, it will not take into account if this is set in config.php, which is probably ok.

Reply:
Ah, good point. I grabbed it first from the poller_cluster_stats but that wouldn't work with cron I think. Is there a place in the db where the polling interval is always set? Or is it just better to "hardcode" the interval?

Reply:
there is no place in the db, which is why you need to use coalesce and set the second value to 300.  that way if set in the DB (which should be the case) it will override the 300.